docker-compose.yml

```diff
@@ -5,9 +5,9 @@ services:
     container_name: simdnode0
     image: "cosmos-sdk/simapp"
     ports:
-      - "26654-26655:26656-26657"
```

Comment:
The ports of the first (node0) should remain identical. Was there a reason they were changed @sahith-narahari ?

Reply:
I remember there was a collision of this with another process, can't recall where though